unit_tests/log/CMakeLists.txt

```diff
@@ -231,7 +236,7 @@ target_compile_definitions(test_log_lsa_utils PRIVATE
 target_include_directories(test_log_lsa_utils PRIVATE
   ${TEST_INCLUDES}
   )
-# target_link_libraries(test_log_lsa_utils fake_thread_utils)
+target_link_libraries(test_log_lsa_utils fake_thread_utils)
```

Comment:
Does test_log_lsa_utils depend on fake_thread_util? It doesn't like it does. If it is does not, better to remove the line completely.